0001-kaa-protocol/README.md

```diff
@@ -6,35 +6,83 @@ editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
 contributors: Alexey Gamov <agamov@cybervisiontech.com>, Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
 ---
 
+- [Introduction](#introduction)
```

Comment:
I know, it's a temporary solution until I find (or someone suggests) a better one.